obal/data/roles/verify_tag/tasks/verify.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Verify tag
+  koji_verify_tag:
+    tag: "{{ tag }}"
+    packages: "{{ hostvars }}"
+    directory: "{{ lookup('env', 'PWD') }}"
+    koji_executable: "{{ koji_executable|default('koji') }}"
+  register: missing
+
+- name: "Packages in Koji but not in git for {{ tag }}"
+  debug:
+    msg: "{{ missing.missing_in_git }}"
+  when: missing.missing_in_git|length > 0
+
+- name: "Packages missing in koji for {{ tag }}"
+  debug:
+    msg: "{{ missing.missing_in_koji }}"
+  when: missing.missing_in_koji|length > 0
+
+- name: "Print packages missing in koji for {{ tag }}"
```

Comment:
This doesn't look entirely appropriate, but I'm not sure what it should say.

Reply:
In what way?

Reply:
It doesn't actually print the packages missing, but rather tells you whether it was all ok. `Summarize koji vs sources status` or something feels closer, but it's still not that great.